types/react-virtualized/dist/es/Table.d.ts

```diff
@@ -404,7 +404,7 @@ export class Table extends PureComponent<TableProps> {
             (params: RowMouseEventHandlerParams) => void
         >;
         onRowsRendered: Requireable<
-            (params: RowMouseEventHandlerParams) => void
+            (params: IndexRange & OverscanIndexRange) => void
```

Comment:
Matches the definition in TableProps: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35756/files#diff-4db0ed7777ff0246bfa9ee9878619ea6R271